### ISSUE TITLE
Getting Null Values from CouchDB (cloudant)

### ISSUE THREAD

**Issue by Yashs21** [biggora/caminte#134]
I am connected to my cloudant account, and I am able to make objects and store it in DB.
But there is one weird issue while retrieving it. 

`User.find({}, function (err, users) {
        console.log(JSON.stringify(users));
        res.send(users);
    });`

For the above query I am getting following response:
`[{"name":null,"bio":null,"age":null,"_rev":null,"id":null},{"name":null,"bio":null,"age":null,"_rev":null,"id":null},{"name":null
,"bio":null,"age":null,"_rev":null,"id":null},{"name":null,"bio":null,"age":null,"_rev":null,"id":null}]`

The number of objects are correct, but the values are not coming up! Please help

**Comment by ravishan110** [comment]
why did you close the issue? How did you solve the problem?

**Comment by Yashs21** [comment]
It happens that they had not much code written for couchdb, when we saw the
implementation of their nano.js, it was incomplete since years! Hence we
decided to code it for ourselves, so closed that issue, but after a point
we stopped using caminte!

I would recommend that if you are using cloudant or couch, just start from
a scratch nodejs app, don't use any library for connection, as their is
none that is well maintained and fully functional . All the best.
